metrics-exporter/src/bin/io_engine/client/replica_stat.rs

```diff
@@ -0,0 +1,79 @@
+use super::ticks_to_time;
+use serde::{Deserialize, Serialize};
+
+/// This stores IoStat information of a replica.
+#[derive(Serialize, Deserialize, Debug, Clone)]
+pub(crate) struct ReplicaIoStat {
+    name: String,
+    entity_id: Option<String>,
```

Comment:
Is there any need in taking replicas with no entity as we can't cross reference them to volumes?

```suggestion
    entity_id: uuid::Uuid,
```

Reply:
Makes sense. We can skip those replicas. However, Dont think we need to convert to UUID as we will label pvc-entity_id ?

Reply:
Right yes, string will do